Cargo.toml

```diff
@@ -261,4 +261,4 @@ lto = true
 
 [profile.profiling]
 inherits = "release"
-debug = "limited"
+debug = true
```

Comment:
`true` is equivalent to `"full"`, `"limited"` — to `1`

Reply:
@Erigara

Reply:
https://doc.rust-lang.org/cargo/reference/profiles.html#debug